### PR TITLE
fix(subgraph): commands so we can deploy on studio as well

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding the ability to deploy on the Subgraph studio as it is now used by Base's testnet!

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: subgraph/bin/thegraph [hunk 1]
```diff
@@ -58,22 +105,32 @@ const deploy = async (network, graphLabel) => {
...
+  throw new Error(
+    `Invalid subgraph configuration for ${network} (${networks[network].subgraph.endpointV2})`
+  )
 }
 
+// Note: the palm network is not supported by
+//
 const choices = Object.keys(networks).filter(
-  (d) => !['networks', 'default'].includes(d)
+  (d) => !['networks', 'localhost', 'default', 'palm'].includes(d)
```
Comment (julien51) [inline subgraph/bin/thegraph:133]: We need to message the team at Palm for the updates...

Comment (julien51) [inline subgraph/bin/thegraph:133]:
```suggestion
  (d) => !['networks', 'default', 'palm'].includes(d)
```

File: subgraph/bin/thegraph [hunk 2]
```diff
@@ -58,22 +105,32 @@ const deploy = async (network, graphLabel) => {
...
+      .split('.')
+      .map((x) => parseInt(x))
+    return deployStudio(`${major}.${minor}.${patch + 1}`, matchStudio[1])
+  }
+  throw new Error(
+    `Invalid subgraph configuration for ${network} (${networks[network].subgraph.endpointV2})`
+  )
 }
 
+// Note: the palm network is not supported by
```
Comment (clemsos) [inline subgraph/bin/thegraph:130]: not supported by the graph?

**Review by clemsos — APPROVED** [review]
lg

File: subgraph/bin/thegraph (hunk 2 above)
Comment (julien51) [reply subgraph/bin/thegraph:130]: Not by the husted service

**Pull request merged**